src/components/templates/Poster/Poster.scss

```diff
@@ -0,0 +1,133 @@
+@import "scss/constants";
+
+$poster-cell-optimal-width: 260px;
+$poster-cell-optimal-height: 195px;
+$poster-grid-gap: 5px;
+$poster-control-btn-height: 65px;
+$poster-control-btn-width: 65px;
+
+.poster {
+  padding: 0 20px 0;
+
+  &__header {
+    display: grid;
+    grid-template-columns: 1fr 1fr 1fr;
+
+    padding: 5px;
+
+    &--middle-cell {
+      display: flex;
+      justify-content: center;
+      align-items: center;
+
+      font-size: 18px;
+      font-weight: 500;
+    }
+    &--right-cell {
+      display: flex;
+      justify-content: flex-end;
+      padding: 0 40px;
+    }
+  }
+
+  &__content {
+    display: grid;
+    grid-template-columns: $poster-cell-optimal-width 1fr 1fr $poster-cell-optimal-width;
+    grid-template-rows: repeat(auto-fill, $poster-cell-optimal-height);
+    gap: $poster-grid-gap;
+
+    grid-template-areas:
+      ".  iframe iframe ."
+      ".  iframe iframe ."
+      ".  iframe iframe ."
+      ".    .     .     ."
+      "listeners listeners listeners listeners";
+  }
+
+  &__control {
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+
+    height: $poster-control-btn-height;
+    width: $poster-control-btn-width;
+    margin: 5px;
+    border-radius: 10px;
+
+    background-color: $dark-grey;
+    cursor: pointer;
+
+    opacity: 0.6;
+
+    &:hover {
+      opacity: 1;
+    }
+
+    &--stop {
+      background-color: $red;
+      color: $white;
+      opacity: 0.8;
+    }
+  }
+
+  &__settings-popover {
+    // This must overwrite default popover height
+    height: 100px !important;
+  }
```

Comment:
Why do we need to use `!important` here? Surely there is a better way?

Reply:
To override the default popover styles

Reply:
There are other ways to override CSS without resorting to `!important`, which is basically a last resort 'no other choice' method.

- https://developer.mozilla.org/en-US/docs/Web/CSS/Specificity

Reply:
Well, not much I can do about it, but I replaced it with an ID in https://github.com/sparkletown/sparkle/pull/1299/commits/edf741ece8f7c71cfb4b0d820cad02be2c3428af

Reply:
ID is even worse tbh.

I thought I already described this above, maybe it was somewhere else in this PR, maybe it was a different PR that was also abusing `!important`.

Check the specificity of the tags you want to override, I would imagine they are likely fairly 'root'/low/something tiny like '1'.

Our BEM classes will also be the same 'root'/low/something tiny like '1'.

So you just want to make the specificity of our override tags high enough to override the existing:

- https://developer.mozilla.org/en-US/docs/Web/CSS/Specificity
- https://developer.mozilla.org/en-US/docs/Learn/CSS/Building_blocks/Cascade_and_inheritance#specificity_2
- https://specifishity.com/

Reply:
done

Reply:
No commit ref for my to check against, so I will assume it's fine, and call it out when I do a full review of the code diff if it isn't.